doc/syscall-interception.md

```diff
@@ -37,6 +37,52 @@ at all.
 
 This can be enabled by setting `security.syscalls.intercept.mknod` to `true`.
 
+### bpf
+The `bpf` system call is used to manage eBPF programs in the kernel.
+Those can be attached to a variety of kernel subsystems.
+
+In general, loading of untrusted eBPF program can be problematic as it
+can facilitate timing based attacks.
+
+LXD's eBPF support is currently restricted to allowing programs use to
+manage devices cgroup entries. To enable it, you need to set both
+`security.syscalls.intercept.bpf` and
+`security.syscalls.intercept.bpf.devices` to true.
+
+### mount
+The `mount` system call allows for mounting both physical and virtual filesystems.
+By default, unprivileged containers are restricted by the kernel to just
+a handful of virtual and network filesystems.
+
+To allow mounting physical filesystems, system call interception can be used.
+LXD offers a variety of options to handle this.
+
+`security.syscalls.intercept.mount` is used to control the entire
+feature and needs to be turned on for any of the other options to work.
+
+`security.syscalls.intercept.mount.allowed` allows specifying a list of
+filesystems which can be directly mounted in the container. This is the
+most dangerous option as it allows the user to feed untrusted data at
+the kernel. This can easily be used to crash the host system or to
+attack it. It should only ever be used in trusted environments.
+
+`security.syscalls.intercept.mount.shift` can be set on top of that so
+the resulting mount is shifted to the UID/GID map used by the container.
+This is needed to avoid everything showing up as nobody/nogroup inside
+of unprivileged containers.
+
+
+The much safer alternative to those is
+`security.syscalls.intercept.mount.fuse` which can be set to pairs of
```

Comment:
How are the pairs set? Comma separated?

Reply:
Yes, doc/instances.md has an example

Reply:
Yep, comma or comma+space as usual for multiple values and it's fsname=fuse-binary for the key=value, there's an example in doc/instances.md.